test/e2e/run_networking_test.go

```diff
@@ -1100,27 +1100,29 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 		pod := "testpod"
 		session := podmanTest.Podman([]string{"pod", "create", "--name", pod})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
+		Expect(session).Should(ExitCleanly())
 
 		net := createNetworkName("IntTest")
 		session = podmanTest.Podman([]string{"network", "create", net})
 		session.WaitWithDefaultTimeout()
 		defer podmanTest.removeNetwork(net)
-		Expect(session).Should(Exit(0))
+		Expect(session).Should(ExitCleanly())
 
 		pod2 := "testpod2"
 		hostname := "hostn1"
 		session = podmanTest.Podman([]string{"pod", "create", "--network", net, "--name", pod2, "--hostname", hostname})
 		session.WaitWithDefaultTimeout()
-		Expect(session).Should(Exit(0))
+		Expect(session).Should(ExitCleanly())
 
 		session = podmanTest.Podman([]string{"run", "--name", "con1", "--network", net, ALPINE, "nslookup", "con1"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(Exit(0))
+		Expect(session.ErrorToString()).To(Equal("nslookup: can't resolve '(null)': Name does not resolve"))
```

Comment:
I find this really icky. Is this really normal and desired?

Reply:
@Luap99 @flouthoc PTAL